### ISSUE TITLE
Do not use more "grep { $v =~ $_ } @v" code than you can help

### ISSUE THREAD

**Issue by azumakuniyuki** [azumakuniyuki/Sisimai#40]
Use regular expression with /x switch instead. Code with a regular expression with /x switch is faster than  a code using grep. To find codes using grep is the following command:
```
% find lib/Sisimai -type f -exec grep grep {} /dev/null \; | grep -i Rx | grep -v ' eq '
```

See https://github.com/azumakuniyuki/perl-benchmark-collection/blob/master/regexp/pipeline-vs-grep.pl

**Comment by azumakuniyuki** [comment]
lib/Sisimai/Reason/*.pm also uses code with grep {...}.

**Comment by azumakuniyuki** [comment]
Fixed at v4.1.10
